src/ducks/settings/AccountsSettings.jsx

```diff
@@ -96,58 +92,61 @@ const AccountsTable = ({ accounts }) => {
           <th className={styles.AcnsStg__actions} />
         </tr>
       </thead>
-      <tbody>{accounts.map(renderAccount)}</tbody>
+      <tbody>
+        {accounts.map(account => (
+          <AccountLine account={account} key={account._id} />
+        ))}
+      </tbody>
     </Table>
   )
 }
 
-class AccountsSettings extends Component {
-  render() {
-    const { t, accountsCollection } = this.props
-
-    if (
-      isCollectionLoading(accountsCollection) &&
-      !hasBeenLoaded(accountsCollection)
-    ) {
-      return <Loading />
-    }
-
-    const sortedAccounts = sortBy(accountsCollection.data, [
-      'institutionLabel',
-      'label'
-    ])
-    const accountBySharingDirection = groupBy(sortedAccounts, account => {
-      return account.shared === undefined
-    })
-
-    const myAccounts = accountBySharingDirection[true]
-    const sharedAccounts = accountBySharingDirection[false]
-
-    return (
-      <div>
-        <AddAccountLink>
-          <Button
-            theme="text"
-            icon={<Icon icon={plus} className="u-mr-half" />}
-            label={t('Accounts.add_bank')}
-          />
-        </AddAccountLink>
-        {myAccounts ? (
-          <AccountsTable accounts={myAccounts} t={t} />
-        ) : (
-          <p>{t('Accounts.no-accounts')}</p>
-        )}
-
-        <h4>{t('Accounts.shared-accounts')}</h4>
-
-        {sharedAccounts ? (
-          <AccountsTable accounts={sharedAccounts} t={t} />
-        ) : (
-          <p>{t('Accounts.no-shared-accounts')}</p>
-        )}
-      </div>
-    )
+const AccountsSettings = props => {
+  const { t } = useI18n()
+  const { accountsCollection } = props
+
+  if (
+    isCollectionLoading(accountsCollection) &&
+    !hasBeenLoaded(accountsCollection)
```

Comment:
https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/utils.js#L30

Reply:
It is taken care of by julien in another PR.